iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -129,9 +129,14 @@
     String getSourceFormat();
 
     /**
-     * Get the state manager of the network.
+     * Get the variant manager of the network.
      */
-    StateManager getStateManager();
+    VariantManager getVariantManager();
+
+    @Deprecated
+    default StateManager getStateManager() {
```

Comment:
Should "default" added on the new method?

Reply:
Here the focus is not the backward compatibility of Network implementations as we known there is only one (NetworkImpl) which is in powsybl core but the API usage compatibility.

Reply:
You cannot be sure of that.. even it is probably true, you should add a default implementation that throw an `UnsupportedOperationException`. This will not break the existing implementation, and a hypothetical implementation would be able to safely remove the implementation of this deprecated method

Reply:
Ok added, also in StateManager